### PR TITLE
reverted to old getScrollPhysics from 1.17 flutter version

### PR DESCRIPTION
**Pull request by dsrenesanse**
## Description
I reverted code to 1.19 candidate 0 to solve overscroll pagination bug, to be honest I was really upsad during last days while solving it from my app side, pls merge it, or fix the problem!

## Related Issues
https://github.com/flutter/flutter/issues/62890

### CODE REVIEW

File: packages/flutter/lib/src/widgets/scroll_configuration.dart [hunk 1]
```diff
@@ -91,12 +91,12 @@ class ScrollBehavior {
     switch (getPlatform(context)) {
       case TargetPlatform.iOS:
       case TargetPlatform.macOS:
-        return _bouncingPhysics;
+        return const BouncingScrollPhysics();
```
Comment (dnfield) [inline packages/flutter/lib/src/widgets/scroll_configuration.dart:94]: This is almost definitely going to break something - the change to use the RangeMaintainingScrollPhysics was made in https://github.com/flutter/flutter/pull/56521 (/cc @hixie) 

You can look at the tests in that PR to help determine what it was fixing.

This PR would need to have at least one test that explains what behavior it's fixing. I'll add some more comments on the issue you've linked.

**Review by dnfield — CHANGES_REQUESTED** [review]
This change, as is, will cause regressions and is missing tests.

**Comment by dnfield** [comment]
I'm going to close this pull request for now to remove it from review queues - if you plan to continue working on it we can certainly re-open. It may be best to continue discussion on the issue first about what test case(s) we're trying to fix before continuing here though.

**Comment by Renesanse** [comment]
I am sorry that I didn't fix exactly the RangeMaintainingScrollPhysics, just spend a lot of time and mental energy to solve this issue before application release. I will try to come back to this pr at my free time =)

**Pull request closed without merging**